jormungandr-integration-tests/src/common/configuration/node_config_model.rs

```diff
@@ -30,7 +30,7 @@ pub struct Peer {
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
-pub struct TopicsOfInterests {
+pub struct Topics {
```

Comment:
I would put these structs in a sub-module `p2p` to mimic the yaml hierarchy.

Reply:
The proposed refactoring seems like a good idea, but I really think this PR is big enough and we should make another one. These ideas could be written down in an issue and discussed there if necessary.